pcsx2/gui/GlobalCommands.cpp

```diff
@@ -173,26 +174,26 @@ namespace Implementations
 	void GSwindow_CycleAspectRatio()
 	{
 		AspectRatioType& art = g_Conf->GSWindow.AspectRatio;
-		wxString arts(L"Not modified");
+		const char *arts = "Not modified";
```

Comment:
The conclusion was next to the variable name (clang-format is also set up to do so, though we currently haven't formatted pcsx2 and GSdx with it).